src/components/Navbar.tsx

```diff
@@ -8,15 +16,14 @@ export const Navbar = () => {
     >
       <div className="container">
```

Comment:
Consider to name this `navbar-container` to improve BEM here
```suggestion
      <div className="navbar-container">
```